nss-day-cohort-44/rare-server-grumpy-reptiles
# Ks server posts

# Description
- Adds get_all_posts 
- Adds get_single_post
- Adds get_post_by_user

## Type of change

# Testing Instructions
```git fetch --all```
```git checkout ks-server-posts```
```code .```
```pipenv shell```
```watchgod request_handler.main```
check each function in postman:
get_all_posts: /posts
get_single_post: /posts/1
get_posts_by_user: /posts?user_id=1

## Issue #
